collections/_article/release-candidate-godot-4-1-2-rc-1.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Release candidate: Godot 4.1.2 RC 1"
+excerpt: "It's long overdue for the second Godot 4.1 patch release! This release candidate contains a number of important fixes, including performance and stability improvements, so let's give it a good test."
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/release-candidate-godot-4-1-2-rc-1.webp
+image_caption_title: "Fisk"
+image_caption_description: "A game by Kristian Nilsen"
+date: 2023-09-22 16:00:00
```

Comment:
```suggestion
date: 2023-09-22 14:00:00
```
Remember it's UTC.

Reply:
Yeah, I just give it an arbitrary estimate when I start :) We managed to get ready quicker!